lib/publiccloud/ssh_interactive_init.pm

```diff
@@ -19,15 +19,13 @@
 
 package publiccloud::ssh_interactive_init;
 use base "consoletest";
-
+use publiccloud::utils;
 use strict;
 use warnings;
 use testapi;
 
 sub post_fail_hook {
-    select_console 'tunnel-console', await_console => 0;
-    send_key "ctrl-c";
-    send_key "ret";
+    select_host_console();
```

Comment:
You are skipping `CTRL+C` here. Not sure if it is a problem. But I guess the reason is, if there is something running on that console, it should be terminated, before typing the next commands.

Reply:
@pdostal wdyt?

Reply:
Ah ok, overlooked this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11711/files#diff-27d3901d8dece7c398b9abdb53c98d17bad7e5c1cf9ed74f14ed9e68a6e6a1c2R34

For me it looks ok. I think an additional `Ctrl+c` and `enter` should not harm.

Reply:
yea. this is what i was thinking too. i was not sure if i had to comment tho to not get people confused of why this is there and get it deleted by accident